data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -2649,7 +2650,7 @@
   "subcategory": "CSC_CHEM_CHEMICALS",
```

Comment:
The byproduct of clean water now doesn't make sense. Either change the proportions back and add result multiplier or remove the byproduct.